src/flightlog.js

```
@@ -1798,47 +1738,81 @@ FlightLog.prototype.getFeatures = function (enabledFeatures) {
     SUPEREXPO_RATES: (enabledFeatures & (1 << 23)) !== 0,
     ANTI_GRAVITY: (enabledFeatures & (1 << 24)) !== 0,
-  };
-};
+  }
+}
 
 FlightLog.prototype.isFieldDisabled = function () {
-  const disabledFields = this.getSysConfig().fields_disabled_mask;
-  const disabledFieldsFlags = {};
+  const disabledFields = this.getSysConfig().fields_disabled_mask
+  const disabledFieldsFlags = {}
```

ESLint

[error] Missing semicolon.

(semi)